packages/mobile/src/app/index.ts

```diff
@@ -1 +1,3 @@
 export * from './App'
+// // Uncomment to run app in storybook mode
+// export { default as App } from '../../.storybook'
```

Comment:
🤔

Reply:
yeah it is super awkward... this is literally what they say to do in their docs: https://github.com/storybookjs/react-native#hideshow-storybook

We should figure out a better option though, whether it's another app etc